testing/surf/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=surf
+pkgver=2.0
+pkgrel=0
+pkgdesc="surf is a simple web browser based on WebKit2/GTK+."
+url="https://surf.suckless.org/"
+arch="all"
+license="MIT"
+depends="xprop"
+makedepends="webkit2gtk-dev"
+options="!check"
+source="$pkgname-$pkgver.tar.bz2::https://git.suckless.org/$pkgname/snapshot/$pkgname-$pkgver.tar.bz2"
```

Comment:
Could you please use the tarball from [dl.suckless.org](https://dl.suckless.org/surf/surf-2.0.tar.gz) instead? We had some problems with tarballs generate by cgit in the past.